grails-app/assets/javascripts/charts2.js

```diff
@@ -1181,7 +1181,7 @@ var taxonomyChart = {
             $recordsLink.html('View records for ' + this.rank + ' ' + this.name);
         }
         else {
-            $recordsLink.html('View all records');
+            $recordsLink.html('<span class="fa fa-list"></span>\nView records');
```

Comment:
\n ? maybe &nbsp; then?

Reply:
It's there just for the spacing not to actually show things in separate rows

Reply:
that's the exact point of this character